iree/compiler/Dialect/HAL/Conversion/FlowToHAL/test/stream_ops.mlir

```diff
@@ -5,7 +5,7 @@ hal.executable @ex0 {
     hal.interface.binding @s0b0, set=0, binding=0, type="StorageBuffer", access="Read"
     hal.interface.binding @s0b1, set=0, binding=1, type="StorageBuffer", access="Read|Write"
   }
-  hal.executable.target "vmla" {
+  hal.executable.target @vmla, "vmla" {
```

Comment:
we could add filter= as a prefix for the filter so that it's clearer what the duplication (in this case) is: `hal.executable.target @vmla, filter="vmla*" {` etc

Reply:
Added `filter=` prefix.